### DISCUSSION TITLE
Multizone audio

### DISCUSSION THREAD

**Discussion by steveman11** [google/oboe#1151] [Q&A] [answered]
Can we mimic multi zone audio in android? Like we can set one zone as input microphone and other zone as speaker.

**Comment by philburk** [comment, marked as answer]
Are you referring to https://source.android.com/devices/automotive/audio/multi_zone/overview

That is normally a configuration set by the Android device manufacturer.

@steveman11 - I am not sure what your goal is. What would be the effect of putting the mic and speaker into separate "zones"?
